config/prow/config.yaml

```diff
@@ -772,6 +773,23 @@ tide:
     - do-not-merge/work-in-progress
     - needs-rebase
     - needs-kind
+  - repos:
+    - kubernetes-sigs/cluster-api
+    labels:
+      - lgtm
+      - approved
+      - "cncf-cla: yes"
+    missingLabels:
```

Comment:
Just making sure here: does cluster-api not need `do-not-merge/cherry-pick-not-approved`?

Reply:
Thx for bringing it up. Currently we don't use this workflow.

We just approve cherry-picks like regular PRs